packages/lib-core/src/utils/random.ts

```diff
@@ -0,0 +1,37 @@
+const getRandomBuffer = () => window.crypto.getRandomValues(new Uint8Array(256));
+
+const isValidCharacter = (char: string) => {
+  const lowercase = 'abcdefghijklmnopqrstuvwxyz'.split('');
+  const uppercase = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'.split('');
+  const numbers = '0123456789'.split('');
+
+  return [...lowercase, ...uppercase, ...numbers].includes(char);
+};
+
+/**
+ * Return a string containing random characters generated via web `crypto` API.
+ *
+ * @see https://deepsource.io/blog/dont-use-math-random/
+ */
+export const getRandomString = (length = 16) => {
```

Comment:
@vojtechszocs couldn't we use a very popular open source package for this such as https://www.npmjs.com/package/uuid?

Reply:
While I consider the above `getRandomString` implementation to be relatively small and solid, I agree with your point about using the existing `uuid` package. I've updated the code in `PluginStore` to use the `uuid` v4 function.